tinker/admin/redirects/__init__.py

```diff
@@ -329,7 +329,7 @@ def redirect_clear(self):
         return self.base.redirect_change()
 
     @requires_auth
-    @route('/public/sftp-publish')
+    @route('/public/sftp-publish', methods=['post'])
```

Comment:
The post will fail, since we use GET requests for the cron. 

Maybe rename to cron-sftp-publish? So it matches the manual one

Reply:
I renamed the method. Should I rename the route too? If I do that we'll have to change the cron-job as well so that it calls the right route (assuming that's how it's being done).

Reply:
yeah that should be easy enough to do. It is managed by puppet, so you can make that change as well.

Reply:
Alright. Done.